packages/mesos-modules/buildinfo.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this to avoid building this package on Linux? This can be done by adding `mesos-modules` to the `exclude` list in `packages/treeinfo.json`.

Reply:
that cause an issue on windows... the above recommendation was suggested by @janisz earlier. there is a separate task in jira to migrate linux modules as well.

Reply:
> Is this to avoid building this package on Linux?
@jongiddy `buildinfo.json` is required to mark directory as a package.